src/style.css

```diff
@@ -0,0 +1,89 @@
+* {
+  margin: 0;
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  color: white;
+  font-size: 100px;
+  text-align: center;
+  line-height: 300px;
+}
+
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 1199px;
```

Comment:
don't hardcode width, it should be able to stretch to the whole width of the viewport